test/clj/rems/api/test_entitlements.clj

```diff
@@ -14,3 +15,39 @@
                        handler)
           body (read-body response)]
       (is (= "unauthorized" body)))))
+
+(deftest entitlements-test
+  (rems.poller.entitlements/run)
+  (let [api-key "42"]
+    (testing "all"
+      (let [data (-> (request :get "/api/entitlements")
+                     (authenticate api-key "developer")
+                     handler
+                     read-body)]
+        (is (= 2 (count data)))))
+
+    (testing "just for alice"
+      (let [data (-> (request :get "/api/entitlements?user=alice")
+                     (authenticate api-key "developer")
+                     handler
+                     read-body)]
+        (is (= 1 (count data)))))
```

Comment:
Maybe do some assert for the content of the data as well?

Reply:
This is an old test that I just moved here. There is not really a good spec on what should be returned so anything goes here. The more interesting tests about content are in the `db/test_applications.clj` so far